### PR TITLE
Cli cleanup 2021 02 16: Cleaned up minor install issues 

### PR DESCRIPTION
**Pull request by tyg**
Issues [432](https://github.com/Agoric/documentation/issues/432) and [435](https://github.com/Agoric/documentation/issues/435)

### CODE REVIEW

**Review by katelynsills — APPROVED** [review]
LGTM, but could probably use another opinion before merging

File: main/getting-started/start-a-project.md
Comment (michaelfig) [inline main/getting-started/start-a-project.md:71]: Just to verify, is this the step for which Xcode is needed, or was it in the `yarn install` step of the Agoric SDK?

**Review by michaelfig — APPROVED** [review]
LGTM.  Just checking if there is somewhere else that the Xcode advice belongs.

**Pull request merged**
